e2e/LoginAuth.json

```diff
@@ -0,0 +1,95 @@
+{
+  "cookies": [
+    {
+      "name": "_ga",
+      "value": "GA1.1.581746587.1692856321",
+      "domain": "localhost",
```

Comment:
@Fahdkassim Can you foresee any scenario where specifying the domain to localhost might fail?

Reply:
The tests have been run in multiple browsers and on different operating systems, and they have passed. I don't think there can be a scenario in which specifying the domain as 'localhost' would cause a failure.